<?xml version="1.0"?>
<launch>
    
    <arg name="fake_sensors" default="True" />

    <include file="$(find multi_car_localization)/launch/fake_car.launch" ns="car0">
        <arg name="car_id" value="0" />
        <arg name="dynamics_model" value="dubins" />
        <arg name="fake_sensors" value="$(arg fake_sensors)" />
    </include>

    <include file="$(find multi_car_localization)/launch/fake_car.launch" ns="car1">
        <arg name="car_id" value="1" />
        <arg name="dynamics_model" value="dubins" />
        <arg name="fake_sensors" value="$(arg fake_sensors)" />
    </include>

    <include file="$(find multi_car_localization)/launch/fake_car.launch" ns="car2">
        <arg name="car_id" value="2" />
        <arg name="dynamics_model" value="dubins" />
        <arg name="fake_sensors" value="$(arg fake_sensors)" />
    </include>

</launch>
